crates/engine/schema/src/builder/coerce/mod.rs

```diff
@@ -244,7 +244,7 @@ impl<'a> InputValueCoercer<'a> {
                 _ => None,
             }
             .map(SchemaInputValueRecord::Boolean),
-            ScalarType::Any => {
+            ScalarType::_Unknown => {
```

Comment:
Why the `_`?

Reply:
well good point, was thinking of Apollo's `_Any` and other types all prefixed with underscore, but here it's not like a client can add a random enum value to `ScalarType`. Removing it